src/pages/beta/[viewType]/key-stages/[keyStageSlug]/subjects/[subjectSlug]/programmes.tsx

```diff
@@ -62,7 +55,7 @@ const ProgrammesListingPage: NextPage<ProgrammeListingPageProps> = (props) => {
                   viewType: "teachers",
                   keyStageSlug,
                 },
-                label: keyStageTitle,
+                label: keyStageTitle ?? "",
```

Comment:
Left out of MV - Will add this back in here - https://www.notion.so/oaknationalacademy/I-want-to-connect-the-programme-listing-MV-to-the-2023-curriculum-API-ebc22f117c434ef4bb917ece325fe1c7